### PR TITLE
Update ROADMAP.md

### PR DESCRIPTION
**Pull request by c4pt0r**

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: docs/ROADMAP.md [hunk 1]
```diff
@@ -42,8 +44,8 @@ This document defines the roadmap for TiDB development.
...
 - [x] LMDB
 - [x] HBase
-- [x] Homemade distributed KV:
-    - [ ] Transactions
-    - [ ] Replicate log using Paxos/Raft
-    - [ ] Auto-Rebalance
+- [x] Homemade distributed KV ([pingcap/tikv](https://github.com/pingcap/tikv)):
+    - [x] Transactions
+    - [x] Replicate log using Paxos/Raft
+    - [x] Scale and Auto-Rebalance
```
Comment (shenli) [inline docs/ROADMAP.md]: "Scale out" it better? BTW I think scaling is a kind of autp-rebalance.

Comment (c4pt0r) [inline docs/ROADMAP.md]: Yep, that makes sense.

File: docs/ROADMAP.md [hunk 2]
```diff
@@ -42,8 +44,8 @@ This document defines the roadmap for TiDB development.
 - [x] RocksDB
 - [x] LMDB
 - [x] HBase
-- [x] Homemade distributed KV:
-    - [ ] Transactions
-    - [ ] Replicate log using Paxos/Raft
-    - [ ] Auto-Rebalance
+- [x] Homemade distributed KV ([pingcap/tikv](https://github.com/pingcap/tikv)):
+    - [x] Transactions
+    - [x] Replicate log using Paxos/Raft
```
Comment (qiuyesuifeng) [inline docs/ROADMAP.md]: Should this be `Replicate log using Raft`?

File: docs/ROADMAP.md [hunk 3]
```diff
@@ -42,8 +44,8 @@ This document defines the roadmap for TiDB development.
...
+    - [x] Scale-out (Auto-rebalance)
```
Comment (shenli) [inline docs/ROADMAP.md]: @ngaut Please review this.

File: docs/ROADMAP.md [hunk 4]
```diff
@@ -42,8 +44,8 @@ This document defines the roadmap for TiDB development.
 - [x] RocksDB
 - [x] LMDB
 - [x] HBase
```
Comment (shenli) [inline docs/ROADMAP.md]: Should we remove this?

File: docs/ROADMAP.md (hunk 3 above)
Comment (ngaut) [inline docs/ROADMAP.md]: LGTM

Comment (ngaut) [inline docs/ROADMAP.md]: We should remove Paxos, just keep raft.

**Comment by shenli** [comment]
@c4pt0r It looks like that you do not sign the CLA.

**Comment by shenli** [comment]
@ngaut @c4pt0r PTAL

**Comment by ngaut** [comment]
LGTM

File: docs/ROADMAP.md (hunk 2 above)
Comment (qiuyesuifeng) [inline docs/ROADMAP.md]: @c4pt0r

Comment (shenli) [inline docs/ROADMAP.md]: I think so.

Comment (c4pt0r) [inline docs/ROADMAP.md]: you're right, just raft

**Comment by c4pt0r** [comment]
LGTM

File: docs/ROADMAP.md (hunk 2 above)
Comment (shenli) [inline docs/ROADMAP.md]: PTAL @qiuyesuifeng

**Pull request merged**